packages/lib/SyncTargetJoplinServer.ts

```
@@ -87,12 +88,12 @@ export default class SyncTargetJoplinServer extends BaseSyncTarget {
 			errorMessage: '',
 		};
 
-		const path = options.path();
-		const protocolErrorMessage = validateUrlProtocol(path);
-		if (protocolErrorMessage) {
-			output.errorMessage = protocolErrorMessage;
-			return output;
-		}
+		const path = options.path().trim();   // trim before validation
```

ESLint

[error] Multiple spaces found before '// trim before...'.

(no-multi-spaces)